crates/compiler/test_mono/generated/dbg_in_expect.txt

```diff
@@ -0,0 +1,12 @@
+procedure Bool.2 ():
+    let Bool.23 : Int1 = true;
+    ret Bool.23;
+
+procedure Test.1 ():
+    let Test.0 : Str = "";
+    dec Test.0;
+    dbg Test.0;
```

Comment:
@JTeeuwissen @folkertdev is this right? I would think the dec should be after the dbg.

Reply:
yeah, we've had issues with this before. maybe a case was missed?

Reply:
I took the liberty to update the code and generated `.txt` on this branch.
This should be the last time expect/dbg cause this kind of problem.